matlab/test/arrow/tabular/tRecordBatch.m

```diff
@@ -494,9 +494,23 @@ function TestIsEqualFalse(testCase)
             % Call isequal with more than two arguments
             testCase.verifyFalse(isequal(rb1, rb2, rb3, rb4));
         end
-
-
+       function testArrowRecordBatchNoArgs()
```

Comment:
Could you please use `UpperCamelCase` for the test name to be consistent with the rest of the test cases in this file? i.e. `TestArrowRecordBatchNoArgs` instead of `testArrowRecordBatchNoArgs`.